src/platform/packages/shared/shared-ux/card/no_data/impl/src/no_data_card.test.tsx

```diff
@@ -7,60 +7,41 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { render as enzymeRender } from 'enzyme';
 import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
nit: this shouldn't be necessary as it is included in global setup. could you please double-check?

Reply:
I see... that's probably because this package overrides default types in tsconfig.json src/platform/packages/shared/shared-ux/card/no_data/impl/tsconfig.json

You can either add `"@testing-library/jest-dom"` to the types or remove the types overrides altogether.